includes/conditions.php

```diff
@@ -39,9 +39,13 @@ public static function compare( $left_value, $right_value, $operator ) {
 			case '!==':
 				return $left_value !== $right_value;
 			case 'in':
-				return false !== array_search( $left_value, $right_value );
+				return in_array( $left_value, $right_value );
```

Comment:
Use a `strict` compare in all `in_array`:
```suggestion
				return in_array( $left_value, $right_value, true );
```